inference-engine/samples/benchmark_app/inputs_filling.cpp

```diff
@@ -91,7 +130,9 @@ void fillBlobImage(Blob::Ptr& inputBlob, const std::vector<std::string>& filePat
                     size_t offset = imageId * numChannels * width * height + (((app_info.layout == "NCHW") || (app_info.layout == "CHW"))
                                                                                   ? (ch * width * height + h * width + w)
                                                                                   : (h * width * numChannels + w * numChannels + ch));
-                    inputBlobData[offset] = static_cast<T>(vreader.at(imageId).get()[h * width * numChannels + w * numChannels + ch]);
+                    inputBlobData[offset] =
+                        static_cast<T>(vreader.at(imageId).get()[h * width * numChannels + w * numChannels + ch]) / static_cast<T>(input_scale[ch]) +
+                        static_cast<T>(input_mean[ch]);
```

Comment:
Please use OpenVINO API for applying values 

https://docs.openvinotoolkit.org/latest/structInferenceEngine_1_1PreProcessChannel.html

Reply:
I reviewed shared API and tested API, but I met exception "not supporting stdScale yet in input".
parameter.cpp / line 115.
                if (fabs(preProcess[c]->stdScale - 1.0f) > 1e-10)
                    IE_THROW() << "not supporting stdScale yet in input " << inputName;
It seems anyvalue can't pass it, Is it true?

Reply:
@vladimir-paramuzov Does it a plugin-specific limitation? This exception in cldnn plugin code.

cc @ilya-lavrenov

Reply:
@nkogteva yes, it's a plugin limitation

Reply:
@hyunback @vladimir-paramuzov 

1. Can you fix it in the GPU plugin?
2. Is there a sense to add one more option on how to apply the preprocessing in benchmark_app: using ov or not? In order to benchmark topology with internal preprocessing as well?

Reply:
@nkogteva  @vladimir-paramuzov 
Thanks for the comment.

Hope to merge this feature first as it is quite handy for our daily work.
What about merging this PR first and follow-up separately for switching into OV preprocessing, including GPU plugin fix?

Reply:
@isanghao For me, it is ok to merge it as is right now.